### ISSUE TITLE
BMEcat 2005.1 Export Error: TypeError: Naugrim\BMEcat\Nodes\Address::setFax(): Argument #1 ($fax) must be of type Naugrim\BMEcat\Nodes\Fax, string given

### ISSUE THREAD

**Issue by sevarozh** [BlackbitDigitalCommerce/pimcore-data-director-bmecat#3]
in Template BMEcat 2005.1 schuld be 

```
if({{ Party 1 Fax }}) {
        $fax = new \\Naugrim\\BMEcat\\Nodes\\Fax();
        $fax->setValue({{ Party 1 Fax }});
        $partyAddress->setFax($fax);
    }
```

instead of 

```
if({{ Party 1 Fax }}) {
        $partyAddress->setFax({{ Party 1 Fax }});
    }

```

Of course for all parts. 

Otherwise the type error occurs.

**Comment by BlackbitDevs** [comment]
Thanks for notifying.
Fixed in 1.4.6

**Comment by sevarozh** [comment]
unfortunately not complete with 2005.1 template: 

Syntax error. There's an extra closing parenthesis here by $fax->setValue().
```
    if({{ Catalog Supplier Address Fax }}) {
        $fax = new \Naugrim\BMEcat\Nodes\Fax();
        $fax->setValue({{ Catalog Supplier Address Fax }}));
        $supplierAddress->setFax($fax);
    }
```
